public/components/layout.js

```diff
@@ -219,18 +219,19 @@ class PointPicker extends HTMLElement {
             <script src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
              integrity="sha256-WBkoXOwTeyKclOHuWtc+i2uENFpDZ9YPdf5Hf+D7ewM="
              crossorigin=""></script>
-            <header title="Use the map below to pan around.  Click to be given the option to use coordinates, 
-                or enter coordinates manually.">
-                Use the map to select coordinates. You may also supply coordinates manually.
-                <br><input disabled id="confirmCoords" type="button" class="button primary" value="Confirm Coordinates" />
+            <header title="Use the map below to pan around">
+                Use the map to draw a geometry. Finalize any shape by double clicking the final point. Subsequent clicks will clear any previous selections.
+                <br><input disabled id="confirmCoords" type="button" class="button primary" value="See Preview" />
             </header>
             <div class="card-body">
-                <div>
-                    <label>Latitude</label>
-                    <input id="leafLat" step=".000000001" type="number" />
-                    <label>Longitude</label>
-                    <input id="leafLong" step=".000000001" type="number" />
+                <div class="row">
+                    <button class="geometry-type-button" style="background-image: url('https://www.svgrepo.com/show/532539/location-pin.svg?size=24') "id="pointBtn" title="Click on the desired location to place a point marker."> </button>
+                    <button class="geometry-type-button" style="background-image: url('https://www.svgrepo.com/show/399231/polyline-pt.svg?size=24');" id="polylineBtn" title="Click to add vertices. Can click the first vertex to finish the shape."> </button>
+                    <button class="geometry-type-button" style="background-image: url('https://www.svgrepo.com/show/399227/polygon-pt.svg?size=24');" id="polygonBtn" title="Clear the map of all selections. Alternatively, click any of the geometry type buttons."> </button>
```

Comment:
I bet this is not the `title` you want for the polygon button.

![image](https://github.com/CenterForDigitalHumanities/geolocator/assets/3287006/98d5ca9d-02a9-49e7-a12f-00a75bb7da3d)